apps/frontend/src/modules/user-courses/slices/user-courses.slice.ts

```diff
@@ -9,12 +9,14 @@ import { add, loadMyCourses, loadUserCourses } from "./actions.js";
 type State = {
 	dataStatus: ValueOf<typeof DataStatus>;
 	myCourses: CourseDto[];
+	totalMyCourses: number;
```

Comment:
maybe this will be more readable
```suggestion
	totalNumberOfMyCourses: number;
```